Innovative Flix/force-app/main/default/objects/Tv_Serie__c/fields/Name__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Name__c</fullName>
+    <caseSensitive>false</caseSensitive>
+    <externalId>false</externalId>
+    <label>Name</label>
```

Comment:
Name this field "Title". It will me more easy to understand. And maybe change API name into "Title__c"

Reply:
Field name 'Title', api name Name__c